_config/config.yml

```diff
@@ -1,22 +1,22 @@
 ---
 Name: foxystripeconfig
 ---
-SiteConfig:
+SilverStripe\SiteConfig\SiteConfig:
   extensions:
-    - FoxyStripeSiteConfig
-Member:
+    - Dynamic\FoxyStripe\ORM\FoxyStripeSiteConfig
+SilverStripe\Security\Member:
   extensions:
-    - CustomerExtension
+    - Dynamic\FoxyStripe\ORM\CustomerExtension
 Page:
   extensions:
-    - FoxyStripePageExtension
-DateField:
+    - Dynamic\FoxyStripe\ORM\FoxyStripePageExtension
+SilverStripe\Forms\DateField:
   default_config:
     showcalendar: true
-FormField:
+FSilverStripe\Forms\ormField:
```

Comment:
Is this supposed to be `SilverStripe\Forms\FormField`?

Reply:
nice catch!